src/hdr_histogram.c

```diff
@@ -50,7 +55,7 @@ static int32_t normalize_index(const struct hdr_histogram* h, int32_t index)
 
 static int64_t counts_get_direct(const struct hdr_histogram* h, int32_t index)
 {
-    return h->counts[index];
+    return atomic_load_explicit((atomic_int_least64_t*)&h->counts[index], memory_order_relaxed);
```

Comment:
I don't believe you need the `(atomic_int_least64_t*)` cast here, given that already the type of `h->counts`.

Reply:
Done